.envrc

```diff
@@ -47,19 +48,23 @@ require() {
     command -v "$1" >/dev/null 2>&1
 }
 
-warn() {
+debug() {
+    if [ "${SENTRY_DIRENV_NOISY}" ]; then
```

Comment:
This is a very opinionated decision and I understand there might be many folks who would prefer the original info statements by default. If someone wants a noisy output they can opt-in.

My ultimate preference would be to delete all debug/info statements unless necessary.